pages/docs/sdk/serve.mdx

```diff
@@ -397,6 +397,50 @@ ff.http('inngest', serve(inngest, [fnA]));
 
 You can run this locally with `npx @google-cloud/functions-framework --target=inngest` which will serve your Inngest functions on port `8080`.
 
+## Framework: Firebase Cloud Functions
+
+Based on the Google Cloud Function architecture, the Firebase Cloud Functions provide a different API to serve functions using `onRequest`:
+
+```typescript
+import { onRequest } from "firebase-functions/v2/https";
+
+import { serve } from "inngest/express";
+import { inngest as inngestClient } from "./inngest/client";
+
+export const inngest = onRequest(
+  serve({
+    client: inngestClient,
+    functions: [/* ...functions... */],
+  })
+);
+```
+
+<Callout>
+
+Firebase Cloud Function requires to configure the `INNGEST_SERVE_PATH` with the custom function path.
```

Comment:
```suggestion
Firebase Cloud Functions require configuring `INNGEST_SERVE_PATH` with the custom function path.
```

"Require" is a transitive verb that needs a direct object to complete its meaning when followed by an infinitive phrase. By changing the phrase to gerund, we are providing a direct object.